modules/doc/content/newsletter/2022_01.md

```diff
@@ -62,6 +62,14 @@ generated by this mesh generator.
       id=quad_peripheral_mesh
       caption=An example peripheral ring (cyan region) generated by `PeripheralRingMeshGenerator`.
 
+### Flux boundary conditions for the Navier Stokes finite volume implementation
+
+Mass, momentum and energy flux boundary conditions were added in order to facilitate
```

Comment:
Could you name these with links to their documentation?

Reply:
will do, adding a commit in next 30 mins at most

Reply:
Force pushing btw
I ll pull right before just in case

Reply:
Looks good - thank you!